templates/paragraphs/paragraph--media-grid.html.twig

```diff
@@ -0,0 +1,14 @@
+{{ attach_library('atomic/image-grid') }}
```

Comment:
I don't see a this library defined, and I don't think we actually need it yet either. (we will when we have the interactive version - with it's javascript). So, I think this can just be deleted for now.